pkg/exchange/bybit/exchange.go

```diff
@@ -315,3 +319,70 @@ func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, since,
 
 	return types.SortOrdersAscending(orders), nil
 }
+
+/*
+QueryTrades queries trades by time range or trade id range.
+If options.StartTime is not specified, you can only query for records in the last 7 days.
+If you want to query for records older than 7 days, options.StartTime is required.
+It supports to query records up to 180 days.
+
+If the orderId is null, fromTradeId is passed, and toTradeId is null, then the result is sorted by
+ticketId in ascend. Otherwise, the result is sorted by ticketId in descend.
+
+** Here includes MakerRebate. If needed, let's discuss how to modify it to return in trade. **
+*/
```

Comment:
Could you also comment that startTime and endTime are inclusive or exclusive?  That will help a lot!